policy/modules/services/corosync.fc

```diff
@@ -6,8 +6,13 @@
 /usr/sbin/corosync	--	gen_context(system_u:object_r:corosync_exec_t,s0)
 /usr/sbin/corosync-notifyd	--	gen_context(system_u:object_r:corosync_exec_t,s0)
 
+/usr/share/corosync/corosync			--	gen_context(system_u:object_r:corosync_exec_t,s0)
+/usr/share/corosync/corosync-notifyd	--	gen_context(system_u:object_r:corosync_exec_t,s0)
```

Comment:
I looked at Fedora 32 and Debian 11 packages and neither have these paths, so this seems to be RHEL7 specific.

Reply:
This could be RHEL7 specific.  Should be be wrapped in a "ifdef distro_redhat" block?

Reply:
Yes.